tests/whimsylib/actor_test.py

```diff
@@ -0,0 +1,45 @@
+from unittest.mock import call
```

Comment:
Out of curiosity, why do we no longer have a `unit/` subdirectory in `tests/`?

Reply:
Nope! We can change the dir structure once we have other kinds of tests, but for now let's keep it simple.